packages/amplify-migration-tests/src/__tests__/migration_tests_v10/__snapshots__/auth-add-all.migration.test.ts.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`amplify migration test auth ...should add auth with max options and work on the latest version 1`] = `
+"Resources
+[~] AWS::Lambda::Function RoleMapFunction 
+ └─ [~] Runtime
+     ├─ [-] nodejs14.x
+     └─ [+] nodejs16.x
+
+Parameters
+[-] Parameter facebookAppId: {\\"Type\\":\\"String\\"}
+[-] Parameter googleClientId: {\\"Type\\":\\"String\\"}
+[-] Parameter amazonAppId: {\\"Type\\":\\"String\\"}
+[-] Parameter appleAppId: {\\"Type\\":\\"String\\"}
+
+Resources
+[~] AWS::Lambda::Function UserPoolClientLambda 
+ └─ [~] Runtime
+     ├─ [-] nodejs14.x
+     └─ [+] nodejs16.x
+[~] AWS::Lambda::Function HostedUICustomResource 
+ └─ [~] Runtime
+     ├─ [-] nodejs14.x
+     └─ [+] nodejs16.x
+[~] AWS::Lambda::Function HostedUIProvidersCustomResource 
+ └─ [~] Runtime
+     ├─ [-] nodejs14.x
+     └─ [+] nodejs16.x
+[~] AWS::Lambda::Function OAuthCustomResource 
+ └─ [~] Runtime
+     ├─ [-] nodejs14.x
+     └─ [+] nodejs16.x
+
+Outputs
+[-] Output GoogleWebClient: {\\"Value\\":{\\"Ref\\":\\"googleClientId\\"}}
+[-] Output FacebookWebClient: {\\"Value\\":{\\"Ref\\":\\"facebookAppId\\"}}
+[-] Output AmazonWebClient: {\\"Value\\":{\\"Ref\\":\\"amazonAppId\\"}}
+[-] Output AppleWebClient: {\\"Value\\":{\\"Ref\\":\\"appleAppId\\"}}
+
+Resources
+[-] AWS::ApiGateway::Deployment DeploymentAPIGWAdminQueries97c92c5d destroy
+[+] AWS::ApiGateway::Deployment DeploymentAPIGWAdminQueries935b745b 
```

Comment:
this snapshot is suspicious, could be a bug ? why is the first resource marked for destroy, and a new one being created

Reply:
this test will continue to fail until the bug is fixed in API repo

Reply:
is it possible to exclude this resource from comparison and create work item to address this later ?